includes/multi-currency/MultiCurrency.php

```diff
@@ -544,6 +548,39 @@ public function update_single_currency_settings( string $currency_code, string $
 		}
 	}
 
+	/**
+	 * Updates the customer currencies option.
+	 *
+	 * @param int $order_id The order ID.
+	 *
+	 * @return void
+	 */
+	public function maybe_update_customer_currencies_option( $order_id ) {
+		$order = wc_get_order( $order_id );
+
+		if ( ! $order ) {
+			return;
+		}
+
+		$currency   = $order->get_currency();
+		$currencies = self::get_all_customer_currencies();
+
+		// Skip if the currency is not a string or if the option is invalid.
+		if ( ! is_string( $currency ) || ! is_array( $currencies ) ) {
```

Comment:
The `$currencies` variable doesn't have the chance to be anything else than an array now after switching to `self::get_all_customer_currencies()`, right? Maybe we should remove it. It being an empty array is also fine for the script to continue as far as I can see.

Reply:
I agree with you. It also appears unnecessary to include the check for `! is_string( $currency )` since orders cannot be created with a non-string currency. Perhaps we should remove the check entirely?

Reply:
I updated the code on https://github.com/Automattic/woocommerce-payments/pull/7143/commits/71ce3fd515e45c3d8617138c916ec6d79f875c51. Thanks for the suggestion and let me know your thoughts on removing both checks.